documentation/modules/post/windows/gather/credentials/viber.md

```diff
@@ -0,0 +1,67 @@
+## Module Description
+
+This post-exploitation module gathers artifacts found on Viber related folders from end users systems.
+
+The list of available artifcts are listed within the module and can be added at anytime. Each artifacts are categorised so that users can specify a category to look for.
+
+
+## Verification Steps
+
+1. Start MSF console
+2. Get a Meterpreter session on a Windows system
+3. use post/windows/gather/credentials/Viber
```

Comment:
lowercase

Reply:
Thank you. Changes were made at 2d34729244d67ac3b010b2b9c734ef79f1ad884f.